### DISCUSSION TITLE
NPM organization name retake policy

### DISCUSSION THREAD

**Discussion by sshuvoo** [community/community#122932] [npm]
I had an npm organization, suppose @example. I deleted the org for a reason, but immediately I tried to create an org with the same name, @example. However, it says, 'The organization name 'example' is not available.' In the meantime, I am assured that no one has taken the name. I badly need this one.

**Comment by chienhsiang-hung** [comment]
Hi @sshuvoo 

According to npm documentation, organizations cannot be renamed from the website or command line interface. To rename an organization, you must manually migrate your existing organization members, teams, and packages to a new organization, then contact npm Support to have the outdated packages unpublished and the previous organization deleted.

This process implies that once an organization is deleted, its name may not be immediately available for reuse. It's possible that npm has a cooldown period to prevent immediate re-registration of organization names, which could be why you're unable to create an organization with the same name right after deletion.

I would recommend contacting npm Support directly to inquire about the specific cooldown period for reusing an organization name and to request assistance with your situation. They should be able to provide you with the most accurate and up-to-date information regarding your issue.

**Reply by sshuvoo** [reply]
Thank you so much for your detailed and helpful response! I truly appreciate you taking the time to explain the process and provide guidance on how to address my issue. Your expertise and willingness to assist are invaluable. Thanks again!

**Reply by santoshlite** [reply]
Hey, I'm facing the same situation! Any idea what is the cooldown period? Were you able to get your org back?

**Reply by sshuvoo** [reply]
Yes, but I don't know the exact cooldown period. I probably tried again one or two months later and got my old name back.

**Reply by santoshlite** [reply]
Okay, thanks!

**Comment by SYNAX07** [comment]
If you're encountering an issue where you're unable to recreate an npm organization with the same name (@example) after deleting it, there are a few reasons why this might be happening:

1. **Deletion Confirmation**: Ensure that the deletion of your previous npm organization (@example) was fully processed and confirmed. Sometimes, it can take some time for the deletion to propagate through the system completely.

2. **Reserved Name**: Certain names on npm may be reserved even after deletion to prevent immediate re-registration. This could be a policy to avoid confusion or misuse of previously deleted names.

3. **System Updates**: Occasionally, there can be delays in the system updating to reflect the availability of a previously used name. Even if you believe the name is available, the system might still be recognizing it as reserved or in a pending state.

Given the urgency of needing this specific organization name, here's what you can do:

- **Contact npm Support**: Reach out to npm Support directly through their official channels. Explain your situation and provide details about the name you want to reuse. They should be able to verify the availability of the name and assist you in re-registering it if it's indeed available.

- **Wait and Retry**: If possible, wait for a little while (could be a few hours to a day) and try again to register the organization with the desired name. Sometimes, delays in system updates can cause this issue to resolve on its own.

- **Consider Alternatives**: While waiting for resolution, you might want to explore alternative organization names that could work for your current needs. This can be a temporary solution until the desired name becomes available again.

It's important to communicate directly with npm's support team for the most accurate and efficient resolution of this issue. They'll have the tools and access needed to investigate why the name (@example) is not immediately available for re-registration and can guide you through the necessary steps to resolve it.
